applications/adv_networking_bench/cpp/testing/benchmark_utils.py

```diff
@@ -156,20 +156,22 @@ def get_tx_throughput(self, port: int) -> float:
         # to get bps, then convert to Gbps
         return (tx_bytes * 8) / (self.exec_time / 1000) / 1e9
 
-    def validate_rx_queue_packets(self, port: int, expected_packets: Dict[int, int]) -> bool:
+    def validate_rx_queue_packets(
+        self, port: int, expected_packets: Dict[int, int], gt: bool = False
+    ) -> bool:
         """
         Validate that RX queues on a specific port received the expected number of packets.
 
         Args:
             port: Port number to validate
             expected_packets: Dictionary mapping queue numbers to expected packet counts {queue: count}
+            gt: If True, validate that the actual packet count is greater than the expected count
```

Comment:
needs https://github.com/nvidia-holoscan/holohub/pull/831/files#r2098692495
```suggestion
        self, port: int, expected_packets: Dict[int, int], allow_greater: bool = False
    ) -> bool:
        """
        Validate that RX queues on a specific port received the expected number of packets.

        Args:
            port: Port number to validate
            expected_packets: Dictionary mapping queue numbers to expected packet counts {queue: count}
            allow_greater: If True, validate that the actual packet count is greater than the expected count
```